lib/tooling/osaca-tool.js

```diff
@@ -29,12 +29,12 @@ import { BaseTool } from './base-tool';
 export class OSACATool extends BaseTool {
     static get key() { return 'osaca-tool'; }
 
-    rewriteAsm(asm) {
-        return asm;
-    }
-
-    writeAsmFile(data, destination) {
-        return fs.writeFile(destination, this.rewriteAsm(data));
+    writeAsmFile(asmParser, asm, filters, destination) {
+        // Applying same filters as applied to compiler outpu
+        var filteredAsm = asmParser.process(asm, filters).asm.reduce(
```

Comment:
Can we make this `const` please? I'm trying to avoid `var`s and their surprising behaviours! Thanks